pypeit/pypeit.py

```diff
@@ -113,7 +113,7 @@ def __init__(self, pypeit_file, verbosity=2, overwrite=True, reuse_masters=False
         if config_specific_file is not None:
             msgs.info(
                 'Setting configuration-specific parameters using {0}'.format(os.path.split(config_specific_file)[1]))
-        spectrograph_cfg_lines = self.spectrograph.config_specific_par(config_specific_file).to_config()
+        spectrograph_cfg_lines = self.spectrograph.config_specific_par(config_specific_file, pypeit_file=pypeit_file).to_config()
 
```

Comment:
@kbwestfall @jhennawi Thank you for your feedback today.
I like the idea of using the `fitstbl` but I'm still not convinced that it will not needed to be created twice.
Once, before the `config_specific_par` (before line 116). And in this case, `self.par` (which is required for `fitstbl`) could be created using the `default_pypeit_par`. Then  `spectrograph_cfg_lines` are generated and `self.par` must be generated again as it's done in line 120, because here makes the merge with the user-proved parsets (giving the preference to the users parsets). And since `self.par` changes, I am guessing that the `fitstbl` should be generated again. Am I making sense?

Reply:
fwiw, the ability to parse lamps out of FITS headers would also be very useful for MMTO Blue Channel. HeNeAr is by far the most commonly used and is what is currently hard-coded. however, ThAr is used with some frequency and FeNe and HgCd are available and used for specific configurations. would be desirable to only include line lists for lamps that are actually turned on for a specific image.